Sources/BraintreeDropIn/Models/BTUIKCardType.m

```diff
@@ -162,8 +162,12 @@ + (NSArray *)allCards
                                                         securityCodeName:BTDropInLocalizedString(CVC_FIELD_PLACEHOLDER)
                                                                 prefixes:@[@"^(5[1-5]|222[1-9]|22[3-9]|2[3-6]|27[0-1]|2720)\\d*"]];
         BTUIKCardType *discover = [[BTUIKCardType alloc] initWithBrand:BTDropInLocalizedString(CARD_TYPE_DISCOVER)
-                                                      securityCodeName:BTDropInLocalizedString(CID_FIELD_PLACEHOLDER)
-                                                              prefixes:@[@"^(6011|65|64[4-9]|622)\\d*"]];
+                                                        securityCodeName:BTDropInLocalizedString(CVV_FIELD_PLACEHOLDER)
```

Comment:
Was this change from `CID_FIELD_PLACEHOLDER` to `CVV_FIELD_PLACEHOLDER` intentional? I think that would be a breaking change for merchants supplying their own localized strings files.

Reply:
Good catch that should not have changed - reverted it